src/actions/generate/templates/cloud-config.yaml

```
@@ -0,0 +1,24 @@
+#cloud-config
+users:
+  - name: wp_user
+    shell: /bin/bash
+    sudo: ['ALL=(ALL) NOPASSWD:ALL'] #revise this
+disable_root: true
+packages:
+  - nginx
+runcmd:
+  - [ ls, -l, / ]
+
+  - [ sh, -xc, "echo $(date) ': hello world!'" ]
+
+  - [ sh, -c, echo "=========hello world=========" ]
+
+  - ls -l /root
+
+  # Note: Don't write files to /tmp from cloud-init use /run/somedir instead.
+
+  # Early boot environments can race systemd-tmpfiles-clean LP: #1707222.
+
+  - mkdir /run/mydir
+
+  - [ wget, "http://slashdot.org", -O, /run/mydir/index.html ]
```

GitHub says the end line is missing
```suggestion
  - [ wget, "http://slashdot.org", -O, /run/mydir/index.html ]

```